client/src/routes/+page.svelte

```diff
@@ -18,34 +150,62 @@
             <label class="visually-hidden form-label" for="location">Location</label>
             <input type="text" class="form-control" id="location" placeholder="Enter a location"
                    aria-label="Location"
-                   aria-describedby="session-configuration">
+                   aria-describedby="session-configuration" bind:value={location}>
             <span class="input-group-append">
                 <img src="{locationImg}" class="input-icon" alt="icon"/>
             </span>
         </div>
 
         <!--CATEGORY INPUT-->
-        <div class="col-lg-4 col-md-12 mb-3">
+        <!-- <div class="col-lg-4 col-md-12 mb-3">
             <label class="visually-hidden form-label" for="category">Food category</label>
             <input type="text" class="form-control" id="category" placeholder="Select a food category"
                    aria-label="Location"
                    aria-describedby="session-configuration">
+            <input class="form-control" id="category" aria-label="Location" aria-describedby="session-configuration" list="foods" type="text" placeholder="Select a food category" autocomplete="on" maxlength="22" bind:value={category}>
+            <datalist id="foods">
+                {#each foodCategories as foodCategory}
+                    <option value={foodCategory}/>
+                {/each}
+            </datalist>
             <span class="input-group-append">
                 <img src="{categoryImg}" class="input-icon" alt="icon"/>
             </span>
-        </div>
+        </div> -->
```

Comment:
Remove commented out code if not needed

Reply:
If categories are not going to be a thing then it is not needed. It is removed